examples/pom.xml

```diff
@@ -45,14 +45,14 @@
         <spring-framework.version>5.2.19.RELEASE</spring-framework.version>
         <spring-boot.version>2.0.9.RELEASE</spring-boot.version>
         <hikari-cp.version>3.4.2</hikari-cp.version>
-        <mysql-connector-java.version>5.1.47</mysql-connector-java.version>
-        <postgresql.version>42.2.5</postgresql.version>
+        <mysql-connector-java.version>8.0.28</mysql-connector-java.version>
```

Comment:
did you test the compatibility for these dependencies ?

Reply:
@taojintianxia to be honest, I don't use shardingsphere - I am just an Apache member concerned about ASF projects relying on old dependencies with publicly disclosed attack vectors

Reply:
my experience is that all of these updates are low impact - that in most scenarios, they can be upgraded without breaking compatibility

Reply:
I see, anyway thanks for your contribution